.github/workflows/build.yaml

```diff
@@ -21,13 +21,31 @@ jobs:
               with:
                   python-version: '>=3.9.0 <4.0.0'
             - run: python3 -m pip install -r ci/cilib/requirements.txt
+
             - uses: docker/login-action@v2
               with:
                   registry: ${{ env.REGISTRY }}
                   username: ${{ github.actor }}
                   password: ${{ secrets.GITHUB_TOKEN }}
 
-            - run: ci/publish-docker --dev
+            - name: Set frontend version
+              run: echo "FRONTEND_VERSION=`ci/version app`" >> $GITHUB_ENV
+
+            - uses: docker/metadata-action@v4
+              with:
+                  images: 'ghcr.io/substra/substra-frontend'
+                  tags: |
+                      type=raw,value=latest,enable={{is_default_branch}}
```

Comment:
I think we discussed that `latest` tag should be set on releases, not the latest commit in main. (I may be mistaken)

Reply:
You're right! I got confused because it was described this way on the asana card. But it makes more sense to set it on releases as otherwise in this context `main` and `latest` would basically mean the same thing.

Reply:
PR updated